runtime/bcverify/vrfyhelp.c

```diff
@@ -1218,3 +1229,32 @@ storeVerifyErrorData (J9BytecodeVerificationData * verifyData, I_16 errorDetailC
 	 */
 	liveStack->pc =  (UDATA)currentPC;
 }
+
+/**
+ * @brief Determine whether the specified field only belongs to the current ROM class.
+ *
+ * @param romClass a pointer to J9ROMClass
+ * @param field a pointer to J9ROMFieldRef
+ * @return TRUE if it exists in the current ROM class rather than superclasses
```

Comment:
Remove "rather than superclasses" as  this code does not deal with that.

Reply:
Removed as suggested above.